e2e/benchmarks/local-benchmark/index.html

```diff
@@ -113,6 +114,9 @@ <h2>TensorFlow.js Model Benchmark</h2>
       if (params.has('task')) {
         task = params.get('task');
       }
+      if (params.has('WEBGPU_PRINT_SHADER')) {
+        untunableFlagsConfig['WEBGPU_PRINT_SHADER'] = params.get('WEBGPU_PRINT_SHADER');
```

Comment:
Can we just tf.env().setFlags() here and avoid all the changes related to untunableFlagsConfig?

Reply:
No, this line https://github.com/tensorflow/tfjs/pull/7523/files#diff-622e3e249bd7bb5327a80ac8034614cdec1761a725466edce4dd57439d734fb8R756 will reset all env flags.